### DISCUSSION TITLE
GOES ID naming convention

### DISCUSSION THREAD

**Discussion by ghost** [opendcs/opendcs#277] [Q&A] [answered]
Are GOES IDs random or do they follow some kind of convention or standard? Is there a way to verify if a random string is a valid GOES ID?

**Comment by MikeNeilson** [comment, marked as answer]
Not in particular. It's just 8 HEX digits. So 0-9 and A-F. That's all controlled by NOAA/NESDIS you can find more documetantion here: https://dcs1.noaa.gov/

Though I'm not sure I've seen anything about a convention other than the HEX digits.
